src/sql/CreateStatement.cpp

```diff
@@ -0,0 +1,71 @@
+
+#include "SelectStatement.h"
```

Comment:
Can you remove the first empty line?
And 2nd: do we need to include select statement here?

Reply:
In the destructor, we try to delete a SelectStatment. If not include the select statement here, the compiler complains:

```
src/sql/CreateStatement.cpp:25:10: error: possible problem detected in invocation of delete operator: [-Werror=delete-incomplete]
   25 |   delete select;
      |          ^~~~~~
src/sql/CreateStatement.cpp:25:10: error: invalid use of incomplete type 'struct hsql::SelectStatement' [-Werror]
In file included from src/sql/SQLStatement.h:6,
                 from src/sql/CreateStatement.h:5,
                 from src/sql/CreateStatement.cpp:1:
src/sql/Expr.h:10:8: note: forward declaration of 'struct hsql::SelectStatement'
   10 | struct SelectStatement;
      |        ^~~~~~~~~~~~~~~
src/sql/CreateStatement.cpp:25:10: note: neither the destructor nor the class-specific operator delete will be called, even if they are declared when the class is defined
   25 |   delete select;
      |          ^~~~~~
cc1plus: all warnings being treated as errors
```

Reply:
The first empty line has been removed.

Reply:
Ah yeah, right. Got it.

Fine from my side. But since my knowledge about the parser is limited, I'd like to get another opinion on it.